llvm/utils/TableGen/InfoByHwMode.cpp

```diff
@@ -35,6 +35,8 @@ ValueTypeByHwMode::ValueTypeByHwMode(Record *R, const CodeGenHwModes &CGH) {
     assert(I.second && "Duplicate entry?");
     (void)I;
   }
+  if (R->isSubClassOf("PtrValueType"))
```

Comment:
I feel like hoisting this TableGen change (as well as its related one in `llvm/Target/Target.td`) to a separate patch will be more ideal, as they're generic changes that might deserve its own test.

Reply:
Since Github PRs don't support stacked diffs, @topperc and I decided that it would make sense for me to submit a multi-commit PR. When I get the LGTM, I'll manually apply each commit to the main branch and close this without squashing and merging. 
Do you agree that this makes sense? If you do, I'll add some tests to the commit.

Reply:
> submit a multi-commit PR.

yeah I'm fine with that. Stacked review has been discussed for a while in the community and I doubt there will be any resolution (either workaround or supports from GH) anytime soon